db.go

```diff
@@ -450,7 +450,8 @@ func (db *DB) Exec(query interface{}, args ...interface{}) (sql.Result, error) {
 	}
 
 	start := time.Now()
-	result, err := db.DB.Exec(querystr, args...)
+	argsConverted := argsConvert(args)
```

Comment:
Is this necessary? I agree that it is good to allow type aliases, but I believe they are already handled in `encodeSQLValue` using `database/sql/driver.DefaultParameterConverter.ConvertValue`. If that isn't working I'd like to understand why and fix it. FYI, `ConvertValue` is doing essentially the same thing as your `argsConvert` method. Try reverting the changes to `db.go` and see if your test still works.